deploy/app.conf.yaml

```diff
@@ -19,10 +19,14 @@ max_size_of_org_signature_pdfa: 204800
 
 cla_platform_url: "${CLA_PLATFORM_URL}"
 
+restricted_corp_email_suffix: ${RESTRICTED_CORP_EMAIL_SUFFIX}
```

Comment:
how to define list of email suffix

Reply:
use the list inline expression like: [qq.com,163.com]